test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -4263,15 +4246,20 @@ def _load_configuration(self, infile, objtype=None):
                     with open(fpath, 'w') as f:
                         f.write(print_hooks)
                     file_qmgr = open(fpath)
-                    cmd = [qmgr]
                     d = self.du.run_cmd(
-                        self.hostname, cmd, stdin=file_qmgr, sudo=True,
-                        logerr=False, level=logging.DEBUG)
+                        self.hostname, [qmgr], stdin=file_qmgr, sudo=True,
+                        logerr=True, level=logging.DEBUG)
```

Comment:
logerr is True by default, did you want to set it to False instead?

Reply:
changed